staff.rb

```diff
@@ -0,0 +1,49 @@
+class Staff
+
+  attr_reader :age, :phase
+  attr_accessor :name
+
+  def initialize(options={})
+    @phase = 3
+    @age = options.fetch(:age, 0)
+    @name = options.fetch(:name, "")
```

Comment:
we have some shared code between `Person` and `Staff` -- how can that be reduced?